config/environment.rb

```diff
@@ -1,5 +1,7 @@
 # Load the rails application
 require File.expand_path('../application', __FILE__)
 
+# require File.expand_path('../initializers/abstract_mysql2_adapter.rb', __FILE__)
```

Comment:
do you still need this and the corresponding file? everything in it is commented out?

Reply:
Currently needed only intermittently - sometimes `db:migrate` causes an error: `Mysql2::Error: All parts of a PRIMARY KEY must be NOT NULL; if you need NULL in a key, use UNIQUE instead`. 
I had to uncomment  it just now to migrate after the rollback. Would you recommend leaving it in its uncommented state in case it's needed later?

Reply:
if everything works with it on, then yes, suggest just leave it un-commented 👍🏽